config/application.rb

```diff
@@ -22,5 +22,9 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.action_view.field_error_proc = Proc.new do |html_tag, instance|
```

Comment:
Unused block argument - `instance`. If it's necessary, use `_` or `_instance` as an argument name to indicate that it won't be used.<br>Use `proc` instead of `Proc.new`.